docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -0,0 +1,91 @@
+configuration files, default setups, and integration with the operating system
+to start as a service.
+
+Installing Trino
+----------------
+
+Download the Trino server RPM package :maven_download:`server-rpm`. Use the
+``rpm`` command to install the package:
+
+.. code-block:: text
+
+    rpm -i trino-server-rpm-*.rpm --nodeps
+
+Installing the :ref:`required Java and Python setup <requirements>` is typically
+managed separately.
+
+Service script
+--------------
+
+The RPM installation deploys a service script configured with ``chkconfig`` so
+that the service can be started automatically on operating system boot. After
+installation, you can manage the Trino server with the ``service`` command:
+
+.. code-block:: text
+
+    service trino [start|stop|restart|status]
+
+* ``start`` Starts the server as a daemon and returns the process ID.
+* ``stop`` Shuts down the server started with either ``start`` or ``run``. Sends
+  the SIGTERM signal.
+* ``restart`` Stops and then starts a running server, or starts a stopped
+  server, assigning a new process ID.
+* ``status`` Prints a status line, either *Stopped pid* or *Running as pid*.
+
+Installation directory structure
+--------------------------------
+
+The RPM package places the various files used by Trino in accordance with the
+Linux Filesystem Hierarchy Standard. This differs from the default tarball
+installation of Trino, where all folders are in the installation directory. For
+example, with a ``tar.gz`` file, configuration files are located by default in the
+``etc`` folder of the installation directory. By contrast, the RPM package
+installation uses ``/etc/trino`` for the same purpose.
+
+The RPM installation places Trino files using the following directory
+structure:
+
+* ``/usr/lib/trino/lib/``: Contains the various libraries needed to run the
+  product. Plugins go in a ``plugin`` subdirectory.
+* ``/etc/trino``: Contains the general Trino configuration files like
+  ``node.properties``, ``jvm.config``, ``config.properties``. Catalog
+  configurations go in a ``catalog`` subdirectory.
+* ``/etc/trino/env.sh``: Java installation path used by Trino.
+* ``/var/log/trino``: Contains the log files.
+* ``/var/lib/trino/data``: This is the data directory.
+* ``/usr/shared/doc/trino``: Contains documentation.
+* ``/etc/rc.d/init.d/trino``: Contains the service scripts for controlling the
+  server process.
+
+RPM-specific configuration settings
+-----------------------------------
+
+The node.properties file requires defining the following two additional
+properties to adjust for the directory structure:
+
+.. code-block:: text
+
+    catalog.config-dir=/etc/trino/catalog
+    plugin.dir=/usr/lib/trino/plugin
+
+Uninstalling
+------------
+
+Uninstalling the RPM is like uninstalling any other RPM, just run:
+
+.. code-block:: text
+
+    rpm -e trino-server-rpm-<version>
+
+Note: During uninstall, any Trino related files deployed will be deleted except
```

Comment:
```suggestion
Note: During uninstall, any Trino related files will be deleted except
```

Reply:
Is this true for config files created by the user after installing the rpm? If not, it should be more clear.

Reply:
I think it deletes the folders basically .. including /etc/trino

Reply:
Also .. "are deleted"

Reply:
Then we need to update this accordingly